### ISSUE TITLE
Mathjax zero-md@{1,2} bug

### ISSUE THREAD

**Issue by cdarve245** [zerodevx/zero-md#86]
Hi,
Thank you for this great tool.
I have been trying to use Mathjax. I was successful using
```
https://cdn.jsdelivr.net/gh/zerodevx/zero-md@1/src/zero-md.min.js
```
However, when I use the newer
```
https://cdn.jsdelivr.net/gh/zerodevx/zero-md@2/dist/zero-md.min.js
```
I get error messages:
```
tex-chtml.js:1 Uncaught TypeError: Cannot set property Package of #<Object> which has only a getter
    at s (tex-chtml.js:1:13801)
    at s (tex-chtml.js:1:13808)
    at s (tex-chtml.js:1:13808)
    at s (tex-chtml.js:1:13808)
    at e.combineWithMathJax (tex-chtml.js:1:14317)
    at tex-chtml.js:1:1144820
    at tex-chtml.js:1:1160983
    at tex-chtml.js:1:1160987
```
Here is the code I added at the end of my script:
```
<script>
    md.addEventListener('zero-md-rendered', () => {
      var el = document.createElement('script');
      el.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js';
      document.head.appendChild(el);
    });
  </script>
```
Is there a way to have Mathjax working with `zero-md@2`?
Thank you!

**Comment by zerodevx** [comment]
Hey, thanks for reporting. I'm going to mark this as a bug - but the root cause probably lies with MathJax. The reason why there are error logs is because `no-shadow` is experimental, and a side effect is `zero-md-rendered` gets fired multiple times - something that I'll try to fix. In the above case, each fire causes the MathJax script to re-run. While the first run works, subsequent runs throw, and that I think is something MathJax should fix. On the browser, the `text-chtml.js` file gets downloaded only once so you won't actually incur the network cost.

**Comment by zerodevx** [comment]
This should be fixed with the latest release `v2.5.1` - the `MathJax` recipe is also updated accordingly: https://zerodevx.github.io/zero-md/recipes/

**Comment by zerodevx** [comment]
Try loading `MathJax` like this:

```html
<head>
  <script>
    // Configure MathJax settings
    MathJax = {
      tex: {
        inlineMath: [
          ['$', '$'],
          ['\\(', '\\)']
        ]
      }
    }
    addEventListener('zero-md-rendered', () => MathJax.typeset())
  </script>
  <!-- Load MathJax library -->
  <script defer src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
  <!-- Load zero-md -->
  <script
    type="module"
    src="https://cdn.jsdelivr.net/gh/zerodevx/zero-md@2/dist/zero-md.min.js"
  ></script>
</head>

<body>
  <!-- Render zero-md into light dom -->
  <zero-md no-shadow>
    <script type="text/markdown">$a_2 + b^2 = d^e$</script>
  </zero-md>
</body>
```
